rule/error_strings_test.go

```diff
@@ -34,7 +34,8 @@ func TestErrorStringsRule_Configure(t *testing.T) {
 		{
 			name:      "Invalid function",
 			arguments: lint.Arguments{"errors."},
-			wantErr:   errors.New("found invalid custom function: errors."),
+			//nolint: revive
```

Comment:
It should be this

```suggestion
			//nolint:revive
```

Please enable `nolintlint` linter via `.golangci.yaml` 

It would lint, the `//nolint` directives